docker-compose.yml

```diff
@@ -15,6 +15,22 @@ services:
       timeout: 5s
       retries: 5
 
+  pgadmin:
+    image: dpage/pgadmin4:8.8
+    container_name: pgadmin
+    restart: always
+    environment:
+      PGADMIN_DEFAULT_EMAIL: admin@local.dev
+      PGADMIN_DEFAULT_PASSWORD: admin
+      PGADMIN_LISTEN_PORT: 5050
+    volumes:
+      - ./docker/postgres/servers.json:/pgadmin4/servers.json
```

Comment:
Is `servers.json` a standard or convention that postgres or the docker image supports? Could you please add a brief comment and link to any relevant documenation?

Reply:
@bryanchriswhite this is something supported by pgadmin; [PGADMIN_SERVER_JSON_FILE](https://www.pgadmin.org/docs/pgadmin4/latest/container_deployment.html#environment-variables)